intel_extension_for_transformers/neural_chat/ui/customized/talkingbot/.env

```diff
@@ -1,2 +1,3 @@
 # AUDIO_URL = 'https://neuralstudio.eglb.intel.com/talkingbot'
-AUDIO_URL = 'http://X.X.X.X:80/v1/talkingbot'
+# AUDIO_URL = 'http://54.87.46.229:80/v1/talkingbot'
+AUDIO_URL = 'http://10.165.59.104:8888/v1/talkingbot'
```

Comment:
Just give a default url? e.g., 127.0.0.1:8888?

Reply:
Yes, you are right!